bin/templates/cordova/Api.js

```diff
@@ -94,7 +95,7 @@ class Api {
     }
 
     addPlugin (pluginInfo, installOptions) {
-        if (!pluginInfo) {
+        if (!pluginInfo || !(pluginInfo instanceof PluginInfo)) {
```

Comment:
This could bite us. `pluginInfo instanceof PluginInfo` would be `false` if we get passed an instance of `PluginInfo` from a `cordova-common` version different from our own, even if it would be API-compatible.